src/NzbDrone.Core/Datastore/Migration/198_parse_titles_from_existing_subtitle_files.cs

```diff
@@ -0,0 +1,68 @@
+using System.Collections.Generic;
+using System.Data;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using Dapper;
+using FluentMigrator;
+using NzbDrone.Core.Datastore.Migration.Framework;
+using NzbDrone.Core.MediaFiles;
+using NzbDrone.Core.MediaFiles.EpisodeImport.Aggregation.Aggregators;
+
+namespace NzbDrone.Core.Datastore.Migration
+{
+    [Migration(198)]
+    public class parse_title_from_existing_subtitle_files : NzbDroneMigrationBase
+    {
+        protected override void MainDbUpgrade()
+        {
+            Alter.Table("SubtitleFiles").AddColumn("Title").AsString().Nullable();
+            Alter.Table("SubtitleFiles").AddColumn("Copy").AsInt32().WithDefaultValue(0);
+            Execute.WithConnection(UpdateTitles);
+        }
+
+        private void UpdateTitles(IDbConnection conn, IDbTransaction tran)
+        {
+            var updates = new List<object>();
+
+            using (var cmd = conn.CreateCommand())
+            {
+                cmd.Transaction = tran;
+                cmd.CommandText = "SELECT \"Id\", \"RelativePath\", \"EpisodeFileId\", \"Language\", \"LanguageTags\" FROM \"SubtitleFiles\"";
+
+                using var reader = cmd.ExecuteReader();
+                while (reader.Read())
+                {
+                    var id = reader.GetInt32(0);
+                    var relativePath = reader.GetString(1);
+                    var episodeFileId = reader.GetInt32(2);
+
+                    var episodeFile = conn.QuerySingle<EpisodeFile>("SELECT * FROM \"EpisodeFiles\" WHERE \"Id\" = @Id", new { Id = episodeFileId });
+
+                    var subtitleTitleInfo = AggregateSubtitleInfo.CleanSubtitleTitleInfo(episodeFile, relativePath);
+
+                    updates.Add(new
+                    {
+                        Id = id,
+                        Title = subtitleTitleInfo.Title,
+                        Language = subtitleTitleInfo.Language,
+                        LanguageTags = subtitleTitleInfo.LanguageTags,
+                        Copy = subtitleTitleInfo.Copy
+                    });
+                }
+            }
+
+            var serializerSettings = new JsonSerializerOptions
```

Comment:
Copy/pasta from another migration?

Reply:
Yea I was wondering about this. It's taken from migration 195 – not sure why it's there either to be honest. If you say I can just get rid of it and it is actually just as unused as it looks that's what I'll do.